libs/sdk-ui-charts/src/highcharts/chartTypes/_util/common.ts

```diff
@@ -166,8 +166,26 @@ export const unwrap = (wrappedObject: unknown): any => {
     return wrappedObject[Object.keys(wrappedObject)[0]];
 };
 
-export function percentFormatter(value: number): string {
-    return isNil(value) ? "" : `${parseFloat(value.toFixed(2))}%`;
+const getNumberWithGivenDecimals = (value: number, decimalNumbers: number) =>
+    decimalNumbers === 0 ? `${Math.round(value)}%` : `${parseFloat(value.toFixed(decimalNumbers))}%`;
+
+export function percentFormatter(value: number, format?: string): string {
+    let numberOfDecimals = 2;
+
+    if (format && !isNil(value)) {
+        const splittedFormat = format.split(".");
```

Comment:
I think that this is fragile. What if you use `,` as a separator and `.` as grouping? Imho you need to inject separator setting into the function and use decimal separator value for this.